deliver/lib/deliver/submit_for_review.rb

```diff
@@ -58,8 +58,21 @@ def wait_for_build(app)
       build = nil
 
       loop do
-        latest_build = find_build(app.latest_version.candidate_builds)
+        # Sometimes candidate_builds don't appear immediately after submittion
+        # Wait for candidate_builds to appear on iTunes
```

Comment:
Small nit pick but can we change `iTunes` to `App Store Connect` in this comment? 🙃

Reply:
@joshdholtz done! :)